src/OrchardCore.Modules/OrchardCore.OpenId/OrchardCore.OpenId.csproj

```diff
@@ -26,6 +26,7 @@
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Recipes.Abstractions\OrchardCore.Recipes.Abstractions.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.ResourceManagement\OrchardCore.ResourceManagement.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Users.Abstractions\OrchardCore.Users.Abstractions.csproj" />
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.Users.Core\OrchardCore.Users.Core.csproj" />
```

Comment:
Why?

Reply:
To be able to read the `LoginSettings` settings which is where it is located.

Reply:
@kevinchalet done!